src/Cn.res

```diff
@@ -0,0 +1,10 @@
+let make = arr => {
+  let result = ref("")
+  for i in 0 to arr->Js.Array2.length - 1 {
+    switch arr->Js.Array2.unsafe_get(i) {
+    | "" => ()
+    | name => result := (i == 0 ? name : result.contents ++ " " ++ name)
```

Comment:
This would result in an extra space at beginning if the first item is an empty string (it's not the only edge case)
what about something like this 
```rescript
let make = arr => {
  let result = ref("")

  for i in 0 to arr->Js.Array2.length - 1 {
    switch (arr->Js.Array2.unsafe_get(i), result.contents) {
    | ("", _) => ()

    | (name, "") => result := name

    | (name, _) => result := result.contents ++ " " ++ name
    }
  }

  result.contents
}

```

Reply:
Great catch, fixed.